docs/licensing/chibios.md

```diff
@@ -3,42 +3,42 @@
 
 **About this document**
 
-This document provides a general overview on the licensing options available when using [ChibiOS](http://chibios.org) as **nanoFramework** RTOS component.
-We've been in touch with the ChibiOS sales team to clarify this as much as possible.
+This document provides a general overview of the licensing options available when using [ChibiOS](http://chibios.org) as the **nanoFramework** RTOS component.
+The nanoFramework team have been in contact with the ChibiOS sales team to ensure accuracy of the following information at time of print (October 2017).
 
-For details and to discuss your particular situation we strongly recommend getting in touch with the ChibiOS sales [here](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
+For details or to discuss your particular situation, we strongly recommend getting in touch with the ChibiOS [sales team](http://chibios.org/dokuwiki/doku.php?id=chibios:licensing:quote).
 
 
-## What exactly is being used from ChibiOS in **nanoFramework**?
+## What exactly is being used from ChibiOS in **nanoFramework firmware**?
 
-**nanoFramework** is built against _unmodified_ ChibiOS sources. It's using:
-* The [HAL](http://chibios.org/dokuwiki/doku.php?id=chibios:product:hal:start), which is released under Apache License 2.0 meaning that it's 100% free also for commercial use.
-* The [RT](http://chibios.org/dokuwiki/doku.php?id=chibios:product:rt:start), which is released under GPL3. See bellow about the licensing options according to your use. 
+**nanoFramework** is built against _unmodified_ ChibiOS sources via a mirror. It's using:
+* The [HAL](http://chibios.org/dokuwiki/doku.php?id=chibios:product:hal:start), which is released under Apache License 2.0 meaning that it's 100% free to use or distribute without royalties for any purpose.
+* The [RT](http://chibios.org/dokuwiki/doku.php?id=chibios:product:rt:start), is released under GPL3. See below for the licensing options which maybe different depending on your particular use. 
 
 
 ## Can I use ChibiOS freely for my hobby or personal development at home?
 
-Yes, using ChibiOS in FOSS project is perfectly fine.
+Yes, using ChibiOS in a Free and Open Source Software project or for personal use is perfectly fine.
 
 
 ## Can I use ChibiOS freely if I'm developing a commercial product?
 
-Here there are several options:
-* Yes, if you are OK with [GPL3](https://www.gnu.org/licenses/gpl.html) licensing terms. Basically you have to open your source code.
-* Yes, if you are OK with ChibiOS publicizing your use of it _and_ you have to clearly mention that your product is using ChibiOS.
+Yes but please be aware of the following options:
+* if you modify the [firmware](nanoframework/nf-interpreter) in any way and you are okay with [GPL3](https://www.gnu.org/licenses/gpl.html) licensing terms. Basically the firmware **MUST** keep your firmware open source.
```

Comment:
I don't think this is correct. (both here and in the line bellow) 
People are OK to change nanoFramework code as they please. We are under Apache 2.0.
ChibiOS licensing is the one that brings that requirement of open sourcing code that is changed.

Reply:
maybe I am getting confused with this bit (Basically you have to open your source code. ), I was trying to get across that you dont have to open source the nanoframework managed app, but may need to open source the firmware if you change it (but dont if you leave as is), or does that only apply to modification of the ChibiOS source?

Reply:
This can be confusing yes. 😄 
You only have to open source modifications that you do to ChibiOS code. Because we are *not* making any modification to ChibiOS source code (that's why the build takes is directly from the mirror) we are good with this.
Perhaps this is what needs to be stressed there. Suggest that "Basically you have to open your source code." is replaced with  the following (or something along those lines...) "Basically you have to open the source code modifications that you make to ChibiOS. If you don't change anything there, this doesn't apply."